serverless/cmd/client/client.go

```diff
@@ -43,11 +44,34 @@ func defaultCacheLocation() string {
 	return fmt.Sprintf("%s/serverless", hd)
 }
 
+// aString is a flag Value which holds multiple strings, allowing the flag to
+// be specified multiple times on the command line.
+type aString []string
+
+func (a *aString) String() string {
+	return fmt.Sprintf("%v", *a)
+}
+
+func (a *aString) Set(v string) error {
+	*a = append(*a, v)
+	return nil
+}
+
+func flagStringList(name, usage string) *aString {
+	r := make(aString, 0)
+	flag.Var(&r, name, usage)
+	return &r
+}
+
 var (
-	logURL     = flag.String("log_url", "", "Log storage root URL, e.g. file:///path/to/log or https://log.server/and/path")
-	cacheDir   = flag.String("cache_dir", defaultCacheLocation(), "Where to cache client state for logs, if empty don't store anything locally.")
-	pubKeyFile = flag.String("public_key", "", "Location of public key file. If unset, uses the contents of the SERVERLESS_LOG_PUBLIC_KEY environment variable.")
-	origin     = flag.String("origin", "", "Expected first line of checkpoints from log.")
+	cacheDir            = flag.String("cache_dir", defaultCacheLocation(), "Where to cache client state for logs, if empty don't store anything locally.")
+	distributorURLs     = flagStringList("distributor_url", "URL identifying the root of a distrbutor (can specify this flag repeatedly")
```

Comment:
```suggestion
	distributorURLs     = flagStringList("distributor_url", "URL identifying the root of a distrbutor (can specify this flag repeatedly)")
```

Reply:
Nit: distributor is misspelled here too.